### PR TITLE
Biz hackweek - Project Showcase

### PR DESCRIPTION
**Pull request by edwardysun**
## Problem solved

Short description of the bug fixed or feature added

---

## PR-Codex overview
This PR introduces a project showcase feature for a dashboard application, enhancing the display of various web3 projects with detailed information and pagination.

### Detailed summary
- Added new project showcase pages in `page.tsx` and `DetailPage` component.
- Implemented project data management with `PROJECT_SHOWCASE_DATA`.
- Introduced UI components like `Card`, `Badge`, and `Button`.
- Included image handling with `next/image` for project visuals.
- Enhanced navigation with pagination in the project listing.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by edwardysun** [comment]
https://thirdweb-lmmdpcbdg.thirdweb-preview.com/project-showcase

**Review by joaquim-verges — APPROVED** [review]
looks good - only thing i would recommend is moving it out of /(dashboard) so you dont have the default header (with the connect/contract/etc tabs)

you can just move it to /app instead so you get the full page

File: apps/dashboard/src/app/(dashboard)/project-showcase/page.tsx [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+      {
+        url: `${getAbsoluteUrl()}/assets/showcase/og_image.png`,
+        width: 1200,
+        height: 630,
+      },
+    ],
+  },
+};
+
+export const dynamic = "force-dynamic";
```
Comment (joaquim-verges) [inline apps/dashboard/src/app/(dashboard)/project-showcase/page.tsx:45]: do you actually need this?

**Comment by edwardysun** [comment]
Sounds good, I believe we actually wanted to have the header. I'll get this in first and we can iterate on that part

**Pull request merged**